saleor/graphql/checkout/tests/mutations/test_checkout_complete_with_transactions.py

```diff
@@ -1640,6 +1653,131 @@ def test_checkout_complete_gift_card_bought(
     assert Fulfillment.objects.count() == 1
 
 
+@patch("saleor.plugins.manager.PluginsManager.order_confirmed")
```

Comment:
an integration marker is missing

Reply:
Basically, it doesn't matter as we are not excluding or running only integration test, I'm also not sure if it's integration test at all.